core/src/com/unciv/ui/objectdescriptions/BuildingDescriptions.kt

```diff
@@ -169,9 +169,6 @@ object BuildingDescriptions {
             textList += FormattedLine(stats.joinToString("/", "{Cost}: "))
         }
 
-        if (requiredTech != null)
-            textList += FormattedLine("Required tech: [$requiredTech]",
-                link="Technology/$requiredTech")
```

Comment:
What we *should* be doing here is 

for tech in requiredTechs
textList += "Required tech: [{tech}]"